latest/toc.md

```diff
@@ -66,6 +66,6 @@ And some books:
 
 Preliminary notes about Dafny integration with code in other programming languages:
    * [Go](Compilation/Go)
-   * [C++](Compilation/Cpp)
+   * [Boogie](Compilation/Boogie)
```

Comment:
Thanks for spotting this dead link ! However, the content of the cpp.md file was moved here:
https://dafny.org/latest/DafnyRef/DafnyRef#2587-c

it's lacking a permanent identifier. Would you be willing to go the extra mile and add the correct link with a permanent identifier?